tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceServiceITest.java

```diff
@@ -341,10 +341,10 @@ public void testStoreAndQueryEndTimeExclude() {
 
         // Query stored trace
         Criteria criteria = new Criteria();
-        criteria.setEndTime(1500);
+        criteria.setEndTime(2);
```

Comment:
These look wrong - the end times are being set precisely - so shouldn't the updated value be 1500000?